### PR TITLE
Improves the description of the installation command

### PR DESCRIPTION
**Pull request by ankur0904**
- **Description:**

 Before: 
`
To install modules needed for the common LLM providers, run:
`

After:
`
To install modules needed for the common LLM providers, run the following command. Please bear in mind that this command is exclusively compatible with the `bash` shell:
`

> This is required for the user so that the user will know if this command is compatible with `zsh` or not.

### CODE REVIEW

File: docs/docs/get_started/installation.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ A lot of the value of LangChain comes when integrating it with various model pro
 By default, the dependencies needed to do that are NOT installed.
 However, there are two other ways to install LangChain that do bring in those dependencies.
```
Comment (baskaryan) [inline docs/docs/get_started/installation.mdx:24]: We're actually planning on deprecating this extras group, i think we probably just want to delete this part of the docs. cc @eyurtsev

Comment (ankur0904) [reply docs/docs/get_started/installation.mdx:24]: @baskaryan Thanks for the review.
Which part of the docs do I need to remove?

Comment (baskaryan) [reply docs/docs/get_started/installation.mdx]: updated!

**Comment by baskaryan** [comment]
thanks @ankur0904!

**Comment by ankur0904** [comment]
@baskaryan Thank you too.

**Pull request merged**